script/os-autoinst-setup-multi-machine

```diff
@@ -16,7 +16,7 @@ ensure_ip_forwarding() {
 }
 
 install_packages() {
-    zypper -n in openvswitch os-autoinst-openvswitch firewalld libcap-progs
+    retry zypper -n in openvswitch os-autoinst-openvswitch firewalld libcap-progs
```

Comment:
I doubt you tested this `retry: invalid option -- 'n'`

Reply:
Should not have tested individual lines outside the script 😿

Reply:
It would be nice if that was correct, though. Then `retry` would be in-line with other "prefix"-commands such as `sudo` and `time`.

Reply:
Not sure what you mean. Those are not used here?

The behavior now is that retry is installed if the command is unavailable. And zypper is retried 3 times if it fails i.e. the default.

Reply:
I think @Martchus means would be nice if github.com/okurz/retry could behave better as in understanding that the parameter "-n" is not meant for retry but the executed command. If one of you has a good idea how https://github.com/okurz/retry/blob/main/retry#L19 could be changed to do that that would be great.

Reply:
https://github.com/okurz/retry/pull/6 should fix that behavior.